presto-spark-launcher/pom.xml

```diff
@@ -47,7 +47,7 @@
         <dependency>
             <groupId>com.facebook.presto.spark</groupId>
             <artifactId>spark-core</artifactId>
-            <scope>provided</scope>
+            <scope>compile</scope>
```

Comment:
It is expected that `spark-core` is provided in runtime because Presto on Spark query is run under Spark environment. What class was not found in your case?

Reply:
org.spark_project.guava.base.Preconditions is one of them.

Reply:
I installed downloaded spark-3.1.1-bin-without-hadoop.tgz and ran Spark with 2 MVs:
VM1: 
- Hadoop ResourceManager, NameNode, DataNode, NodeManager
- Hive: HCatalog server
- Spark: Master, Worker

VM2:
- Hadoop: SecondaryNameNode, DataNode, NodeManager
- Spark: Worker